man/io_uring_prep_setxattr.3

```diff
@@ -0,0 +1,65 @@
+.\" Copyright (C) 2023 Rutvik Patel <heyrutvik@gmail.com>
+.\"
+.\" SPDX-License-Identifier: LGPL-2.0-or-later
+.\"
+.TH io_uring_prep_setxattr 3 "January 23, 2023" "liburing-2.3" "liburing Manual"
+.SH NAME
+io_uring_prep_setxattr, io_uring_prep_fsetxattr \- prepare a request to set an 
+extended attribute value
+.SH SYNOPSIS
+.nf
+.B #include <liburing.h>
+.PP
+.BI "void io_uring_prep_setxattr(struct io_uring_sqe *" sqe ","
+.BI "                            const char *" name ","
+.BI "                            const char *" value ","
+.BI "                            const char *" path ","
+.BI "                            int " flags ","
+.BI "                            unsigned int " len ");"
+.PP
+.BI "void io_uring_prep_fsetxattr(struct io_uring_sqe *" sqe ","
+.BI "                             int " fd ","
+.BI "                             const char *" name ","
+.BI "                             const char *" value ","
+.BI "                             int " flags ","
+.BI "                             unsigned int " len ");"
+.fi
+.SH DESCRIPTION
+.PP
+The
+.BR io_uring_prep_setxattr(3)
+function prepares a request to set an extended attribute value. The submission 
+queue entry 
+.I sqe
+is setup to set the
+.I value
+of the extended attribute identified by
+.I name
+and associated with the given
+.I path
+in the filesystem with modifier flags
+.I flags.
+The
+.I len
+argument specifies the size (in bytes) of
+.I value.
+
+.BR io_uring_prep_fsetxattr(3)
+is identical to 
+.BR io_uring_prep_setxattr(3),
+only the extended attribute is set on the open file referred to by
+.I fd
+in place of
+.I path.
```

Comment:
This dot after "`path`" should not be underlined, it should be a roman
font.

That means, don't write this:
```
.I path.
```

But write this:
```
.IR path .
```

Just like what we have here (this is an example where you have a dot after mentioning a variable):
https://github.com/axboe/liburing/blob/b82c5b77cd34edbfdb526b83ac1bad1935647299/man/io_uring_prep_recv.3#L38

Side note:
`.I` typesets in italics (but terminals show that as underline), and `.R`
in what troff calls the roman font, and the rest of us call the normal
font.

You can combine these as `.IR` (the first argument is italic, the second is
roman).

Reply:
I see. Thanks for explaining it. I was just looking at the terminal whether it was formatted properly or not.